app/model.js

```diff
@@ -119,6 +119,50 @@ const fetchArticleById = (params) =>
     }
 }
 
+const addUserComment = (params, body) =>
+{
+    if(!/[0-9]+/g.test(params))
+    {
+        return Promise.reject({status: 400, msg: 'invalid article ID: not a number'})
+    }
+    return db.query
+    (
+        `
+        SELECT *
```

Comment:
params should be article_id - this is confusing if you don't know what has been passed from the controller.